subsys/power/x86_non_dsx.c

```diff
@@ -244,11 +239,24 @@ static void power_pass_thru_handler(const char *in_signal,
 	}
 }
 
+void espi_bus_reset(void)
+{
+	/* If SOC is up toggle the PM_PWRBTN pin */
+	if (gpio_get_level(GPIO_NET_NAME(PCH_EC_SLP_SUS_L))) {
+		LOG_INF("Toggle PM PWRBTN");
+
+		gpio_set_level(GPIO_NET_NAME(EC_PCH_PWR_BTN_ODL), 0);
```

Comment:
This is where we should be using device tree to reference the gpios that the power sequence node listed